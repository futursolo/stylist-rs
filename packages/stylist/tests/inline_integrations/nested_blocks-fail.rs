fn main() {
    let _ = stylist::css! {
        .outer {
            .inner {
                background-color: red;
            }
        }
    };
}
