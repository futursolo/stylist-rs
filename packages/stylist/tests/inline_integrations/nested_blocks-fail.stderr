error: Can not nest qualified blocks (yet)
 --> $DIR/nested_blocks-fail.rs:4:13
  |
4 |             .inner {
  |             ^^^^^^
